SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -120,34 +123,28 @@ EntityComponentUpdate SpatialLoadBalanceEnforcer::ConstructAclUpdate(const Worke
 		ComponentIds.AddUnique(RequiredComponentId);
 	}
 
-	// Get the client worker ID net-owning this Actor from the NetOwningClientWorker.
-	const PhysicalWorkerName PossessingClientId =
-		NetOwningClientWorkerComponent.WorkerId.IsSet() ? NetOwningClientWorkerComponent.WorkerId.GetValue() : FString();
-
-	const FString& WriteWorkerId = FString::Printf(TEXT("workerId:%s"), **DestinationWorkerId);
-
-	const WorkerAttributeSet OwningServerWorkerAttributeSet = { WriteWorkerId };
-
 	for (const Worker_ComponentId ComponentId : ComponentIds)
 	{
 		switch (ComponentId)
 		{
 		case SpatialConstants::HEARTBEAT_COMPONENT_ID:
 		case SpatialConstants::CLIENT_RPC_ENDPOINT_COMPONENT_ID_LEGACY:
 		case SpatialConstants::CLIENT_ENDPOINT_COMPONENT_ID:
-			Acl.ComponentWriteAcl.Add(ComponentId, { { PossessingClientId } });
+			AuthorityDelegationComponent.Delegations.Add(ComponentId, ClientWorkerPartitionId);
 			break;
-		case SpatialConstants::ENTITY_ACL_COMPONENT_ID:
-			Acl.ComponentWriteAcl.Add(ComponentId, { SpatialConstants::UnrealServerAttributeSet });
+		case SpatialConstants::POSITION_COMPONENT_ID:
+		case SpatialConstants::INTEREST_COMPONENT_ID:
+		case SpatialConstants::AUTHORITY_DELEGATION_COMPONENT_ID:
+		case SpatialConstants::METADATA_COMPONENT_ID:
+		case SpatialConstants::PERSISTENCE_COMPONENT_ID:
```

Comment:
Are we skipping these because they're handled by a component set with a different ID? If so, why would these be in the delegations to begin with?

Reply:
This list is built from component presence, which currently needs to call out these individual components as they're the only ones that use a component set (instead of just being wrapped in a component set with the same id). This is updated in @alastairdglennie subsequent component presence removal pr.